docs/development/quick-start.md

```diff
@@ -18,7 +18,7 @@ Start by cloning the Pharos repository:
 $ git clone git@github.com:ithaka/pharos.git
 ```
 
-Pharos supports the current long-term support (LTS) version of Node.js—currently version 20. Node can be installed from [its website](https://nodejs.org/en/), with [nvm](https://github.com/nvm-sh/nvm#install--update-script), or with [asdf](https://asdf-vm.com/) and its [Node.js plugin](https://github.com/asdf-vm/asdf-nodejs).
+Refer to [.tool-versions](../../.tool-versions) for the latest node version compatible with Pharos. Node can be installed from [its website](https://nodejs.org/en/), with [nvm](https://github.com/nvm-sh/nvm#install--update-script), or with [asdf](https://asdf-vm.com/) and its [Node.js plugin](https://github.com/asdf-vm/asdf-nodejs).
```

Comment:
I feel like we lose a little bit of accountability by removing the statement that "Pharos supports the current LTS release of Node.js" - It seems partially a statement of intent?

Reply:
Yeah I would like to keep the statement of intent and remove only the specific version.

Reply:
Could we add renovate and a statement about using that to stay on top of dependencies to recover accountability while also being more accurate about what we currently support as this change intends to do?

Reply:
I'm all for adding Renovate, but that feels like a separate thing a bit? Since this is the quickstart, it's nice to be able to tell people just getting started we use the latest LTS version as our baseline.

What about something like:
> "Pharos is committed to supporting the current long-term support (LTS) version of Node.js. Refer to the [.tool-versions](../../.tool-versions) file for the latest version compatible with Pharos."

Reply:
Good suggestion. Thanks!

Reply:
FWIW there is an open issue for using Dependabot for automated dependency updates, which I would currently still be inclined to use so that Pharos is not using proprietary infrastructure. But we should discuss it more there.